doc/modules/preprocessing.rst

```diff
@@ -830,6 +830,72 @@ lexicon order.
    >>> enc.infrequent_categories_
    [array(['b', 'c'], dtype=object)]
 
+.. _target_encoder:
+
+Target Regressor Encoder
+------------------------
+
+.. currentmodule:: sklearn.preprocessing
+
+The :class:`TargetEncoder` uses target mean groupby the
```

Comment:
I feel saying "per value of the category feature" is going to be more clear? Or "grouped by"? `groupby` is not a common usage within ml, right?